### PR TITLE
added SCUTTLE robot

### PR DESCRIPTION
**Pull request by ansarid**

### CODE REVIEW

File: _posts/2020-06-06-scuttle.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- 'vacuum'
+- 'wheeled'
+- 'wifi'
+- 'wireless'
+twitter_text: Checkout the SCUTTLE
+website: https://mxet.github.io/SCUTTLE/
+wiki_homepage: 
+---
+
+TODO Fill in the long description/article here.
```
Comment (tfoote) [inline _posts/2020-06-06-scuttle.md:62]: The content for this page is only the preexisting TODO

File: _posts/2020-06-06-scuttle.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+layout: post
+title: SCUTTLE
+date: 2020-06-06
+description:
+icon: /assets/img/robots/scuttle/icon.png
+image: /assets/img/robots/scuttle/icon.png
+introduction: A low-cost open-source payload-capable robot for hobby and university research.
+main-class: 'ground'
+color: '#FFFFFF'
+tags:
```
Comment (tfoote) [inline _posts/2020-06-06-scuttle.md:11]: Please use a limited number of tags. Preferrably reusing existing tags https://robots.ros.org/tags/

File: _posts/2020-06-06-scuttle.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+- '50kg payload'
+- 'Affordable'
+- 'affordable'
+- 'auto charging dock'
+- 'automation'
+- 'autonomous'
+- 'autonomus'
```
Comment (tfoote) [inline _posts/2020-06-06-scuttle.md:18]: Make sure not to have duplicate typos

File: _posts/2020-06-06-scuttle.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- 'single-board-computer'
+- 'slam'
+- 'teaching robot'
+- 'vacuum'
+- 'wheeled'
+- 'wifi'
+- 'wireless'
+twitter_text: Checkout the SCUTTLE
+website: https://mxet.github.io/SCUTTLE/
+wiki_homepage:
```
Comment (tfoote) [inline _posts/2020-06-06-scuttle.md:59]: There's no required wiki documentation.

**Comment by ansarid** [comment]
My mistake. I'll fix those.

**Comment by tfoote** [comment]
I'm going to close this due to no response. @ansarid Please feel free to update your branch and open a new PR for another review.

**Pull request closed without merging**